src/pypromice/L0toL1.py

```diff
@@ -137,22 +139,31 @@ def _flagNAN(ds, flag_file=None):
     ds : xr.Dataset
         Level 0 data with flagged data
     '''
-    # flag_file = "./data/flags/" + ds.attrs["station_id"] + ".csv"
-    if flag_file is None:
-        print('Flag file not given. No data flagged')
+    ds = ds_in.copy()
```

Comment:
I think the URL grabbing could be simply done with `pd.read_csv` from URL, rather than downloading the .csv files.
```
# Construct URL
flag_url = "https://raw.githubusercontent.com/GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues/master/flags/" + ds.attrs["station_id"] + ".csv"

# Attempt to retrieve URL csv as DataFrame
try:
    df = pd.read_csv(flag_url,
                                comment="#", 
                                skipinitialspace=True,
                                 ).dropna(how='all', axis='rows')
except:
    print('Could not find flag file')
```    
Then there is no need for an option of retrieving flag files from a local copy.

Reply:
I'm OK with that, although I have a preference for the local copy.